client/src/components/App.jsx

```diff
@@ -1,10 +1,94 @@
 import React, { Component } from 'react';
 
 class App extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      imgBoxesStatus: {
+        one: "inactive",
+        two: "inactive",
+        three: "inactive",
+        four: "inactive",
+        five: "inactive"
+      }
+    };
+
+    this.mouseOverHandler = this.mouseOverHandler.bind(this);
+    this.mouseOutHandler = this.mouseOutHandler.bind(this);
+  }
+
+  mouseOverHandler(event) {
+
+    let eventTargetName = event.target.getAttribute('name');
+
+    let tempObj = {};
```

Comment:
would name this something more descriptive than tempObj, maybe identifyActivePicture or something.